app/routes/projects/manageMembership.tsx

```diff
@@ -0,0 +1,29 @@
+import { redirect, type ActionFunction } from "@remix-run/server-runtime";
+import { multipleProjectsValidator } from "~/core/components/MembershipModal";
+import { updateProjectActivity } from "~/models/project.server";
+
+
+export const action: ActionFunction = async ({ request }) => {
+    const result = await multipleProjectsValidator.validate(await request.formData());
+    const projects = result.data?.projects;
+    
+    if(projects){
+        try {
+            await updateProjectActivity(projects);
+            return redirect("/projects");
+          } catch (e) {
+            throw e;
+          }
```

Comment:
I think we have some indent differences here as well, in some cases we are using 4 spaces and in other 2 spaces. I think the standard we've been using is 2 spaces, please let me know if I'm wrong about this.